pkg/webui/console/components/gateway-map/index.js

```diff
@@ -0,0 +1,51 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import MapWidget from '../../../components/map/widget/'
+
+export default class GatewayMap extends React.Component {
```

Comment:
please update pr title and description to include changes to the gateway pages or do it in a separate pr